contracts/WROSE.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+contract WROSE {
```

Comment:
```suggestion
/// This WROSE allows meta-withdrawals to accounts that do not have gas.
/// Token transfers are not private because storage operators can see which accounts are active.
contract WROSE {
```

Reply:
Added comments